package.json

```diff
@@ -368,7 +369,7 @@
         "dotenv": "^16.0.0",
         "open": "^8.0.4",
         "semver": "^7.3.5",
-        "vscode-azureextensionui": "file:../vscode-azuretools/ui/vscode-azureextensionui-0.50.0.tgz",
+        "vscode-azureextensionui": "^0.50.0",
```

Comment:
This is the newish old SDK, do we want to be using the split one instead?

Reply:
Yeah, I'm working on that on another branch now